### DISCUSSION TITLE
Authentication templates

### DISCUSSION THREAD

**Discussion by nssnl** [netflie/whatsapp-cloud-api#109] [General]
New authentication templates have been released. I just created such a template for sending OTP's.
When trying to send it, I'm getting an error. Seems that `"category": "AUTHENTICATION"` should be posted along with the request. It this possible?
https://developers.facebook.com/docs/whatsapp/business-management-api/authentication-templates/

**Comment by aalbarca** [comment]
@nssnl Thanks for reporting this new feature. I will try to develop it but the lack of financing for this project means that the times can be extended.

**Comment by nssnl** [comment]
WhatsApp sent a mailing yesterday, saying that they have extended the deadline to comply with the authentication templates. It's now April 1, 2024 (previously Oct 2 2023). That gives a bit more time, though I guess this only applies to existing authentication templates, created prior to June 1 2023.

**Comment by nssnl** [comment]
This discussion can be closed, since the authentication template is no longer needed `"category": "AUTHENTICATION"`. It is just a regular template, see: https://developers.facebook.com/docs/whatsapp/cloud-api/guides/send-message-templates/auth-otp-template-messages (don't know when they changed it)
